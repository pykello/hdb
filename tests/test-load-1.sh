#!/bin/sh

hdb_port_string=`./launch-hdb.py 2 10 100`
read -a ports <<< $hdb_port_string

echo "Loading ..."
./load-hdb.py ${ports[0]} < ./data/graph.1

echo "Getting statistics ..."
./stat-hdb.py ${ports[0]}

./stop-hdb.py ${ports[0]}

