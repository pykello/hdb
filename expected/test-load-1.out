Loading ...
Loaded 6
Getting statistics ...
${ports[0]}
Nodes: 2 (20%)
Relationships: 2 (2%)
Max Degree: 1
${ports[1]}
Nodes: 2 (20%)
Relationships: 4 (4%)
Max Degree: 3
Shutting down (localhost, ${ports[0]}) ...
OK.
Shutting down (localhost, ${ports[1]}) ...
OK.
